README.md

```diff
@@ -47,6 +47,9 @@ you enter it.
 Getting rid of the persistent container is done with `make dev-clean`, whereas
 removing its image is done using `make dev-clean-image`.
 
+Alternatively you can start a container with the service running with `make dev-service`. This rule is intended for when you want to test
```

Comment:
You might want to start the new phrase in a new line (ie. it does not matter when visualizing the markdown, but when reading the source ASCII text file this line stands out).

Reply:
I'll edit the README.md file so it is more easily readable when not using a Markdown visualizer. I'll set the limit line of the lines I write to 80 characters as I see it seems is being done